relayer/crates/starknet-chain-components/src/impls/queries/ack_commitment.rs

```diff
@@ -120,14 +117,9 @@ where
             proof_bytes: signed_bytes,
         };
 
-        // assert sha256 hash of SUCCESS_ACK is stored ack_bytes
-        // asset_eq!(ack_bytes, SUCCESS_ACK);
-
-        // FIXME: we can't send `ack_bytes` as it is stored after hashing.
-        // We should query block event to get the original ack_bytes.
-        //
-        // For now, we just return the SUCCESS_ACK as ack_bytes.
+        // `ack_bytes` is stored after hashing.
+        // query block event is required to get the original ack_bytes.
 
-        Ok((SUCCESS_ACK.to_vec(), dummy_proof))
+        Ok((ack_bytes, dummy_proof))
```

Comment:
just wanted to confirm that this method now expects acknowledgment commitment bytes (hashed) at hermes-sdk?

Reply:
Currently the ack bytes are not used, only the proof https://github.com/informalsystems/hermes-sdk/blob/main/crates/chain/chain-components/src/impls/payload_builders/packet.rs#L101